src/pages/home/report/ReportActionsView.js

```diff
@@ -124,6 +131,46 @@ class ReportActionsView extends React.Component {
         this.setState({refetchNeeded});
     }
 
+    /**
+     * Gets the next attachment based on the current action passed into the function
+     * and the direction being moved (to he left or to the right)
+     *
+     * @param {ReportActionPropTypes} action
+     * @param {Boolean} toRight
+     *
+     * @returns {ReportActionPropTypes}
+     */
+    getNextAttachment(action, toRight) {
+        const actions = Object.values(this.sortedReportActions).reverse();
```

Comment:
consider using [underscore chain](https://underscorejs.org/#chain) to clean up the logic here such that you get exactly the action you want and then return `nextAttachment.action` that the places calling this have the action object directly that doing `.action` each time.